src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,87 @@
+.menu {
+  background-color: #e5e5e5;
+  &__content {
+
```

Comment:
When menu is open full page scroll works.
In the video lesson you can see how to avoid this

![Screenshot_1](https://user-images.githubusercontent.com/89162624/208913248-bd9a2731-c996-41a6-8ddf-f8982e7a1067.png)